CHANGELOG.md

```diff
@@ -22,9 +22,10 @@ Changes
 * [UI]: Updated icons for datatables sorting and metadata importer.
 * [UI]: Fixed bug where users could not update their email subscriptions to projects. (19.01.2)
 * [UI/Developer]: Fixed issue where search for member to add to group resulted in no results found when search term contained capital letters.
-* [Documentation]: Added information on fixing ONLY_FULL_GROUP_BY sql error to the administrator faq docs.
+* [Documentation]: Added information on fixing `ONLY_FULL_GROUP_BY` sql error to the administrator faq docs.
 * [UI]: User on a remote project with a project role of manager has the ability to assign user groups to the project.
 * [UI]: Fixed bug preventing associated projects from being loaded into the project samples table. (19.01.3)
+* [Developer]: Moved FastQC results out of database to filesystem for a big reduction in database size and performance.  See https://irida.corefacility.ca/documentation/upgrades/#1904 for more information. 
```

Comment:
I believe this link should be <https://irida.corefacility.ca/documentation/administrator/upgrades/#1904>.

Reply:
You're right.  That means it was also broken in `UPGRADING.md`.  Fixed both in 0bf2c34

Reply:
Awesome. Thanks.